### PR TITLE
fix cmake BOOL type

### PR DESCRIPTION
**Pull request by bd4**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,4 @@
 build_*
 install_*
 source.me
+perfstubs_api/config.h
```
Comment (germasch) [inline .gitignore]: This looks  suspicious to me -- why is this needed? If config.h gets created in the source directory, that's a bug that should be fixed. But from looking at `CMakeLists.txt`, it looks to me like it'll correctly get created in the build directory. But then where did it come from?

Comment (bd4) [reply .gitignore]: Oops you are correct... either I created it, or it's from an earlier iteration of the build files. I'll remove this from the PR.

**Comment by khuck** [comment]
I've been seeing that message and wondering why it was happening, thanks for fixing it.  I should have read the CMake documentation a little more carefully.
